### PR TITLE
Add Installers workflow

### PR DESCRIPTION
**Pull request by ErebusZ**

### CODE REVIEW

File: .github/workflows/mac_installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+on:
+  release:
+    types: [published]
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
```
Comment (deadly-panda) [inline .github/workflows/mac_installer.yaml:10]: Why false ?

Comment (ErebusZ) [reply .github/workflows/mac_installer.yaml:10]: originally I had one workflow for both mac and ubuntu, this is no longer the case. I ll remove it.

**Review by benyissa — CHANGES_REQUESTED** [review]
We don't need a matrix strategy since there is only one configuration.

File: .github/workflows/mac_installer.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies.
+      run: |
+        python -m pip install --upgrade pip
+        pip install .
+        pip install .[scanner]
+        pip install .[agent]
+
+    - run: |
```
Comment (benyissa) [inline .github/workflows/mac_installer.yaml:29]:
```suggestion
    - name: Create DMG file
    - run: |
```

Comment (3asm) [reply .github/workflows/mac_installer.yaml:29]: +1

**Review by 3asm — APPROVED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**